app/fiilee.py

```diff
@@ -0,0 +1,15 @@
+import json
+
+
+class Config:
+    def __init__(self, file_name: str="C:\\Users"
+                                 "\\Користувач"
+                                 "\\enviropment_skill"
+                                 "\\py-shop-trip"
+                                 "\\app"
+                                 "\\config.json") -> None:
+        with open(file_name, "r") as f:
```

Comment:
It's a good practice to include error handling when opening files. Consider adding a try-except block to handle potential exceptions, such as FileNotFoundError or json.JSONDecodeError.